planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/data_structs.hpp

```diff
@@ -313,6 +315,25 @@ struct LanesPolygon
   std::vector<lanelet::BasicPolygon2d> preceding_target;
 };
 
+struct MinMaxValue
+{
+  double min{std::numeric_limits<double>::infinity()};
+  double max{std::numeric_limits<double>::infinity()};
+};
+
+struct TransientData
+{
+  MinMaxValue acc;
+  MinMaxValue lane_changing_length;
+  MinMaxValue current_dist_buffer;
+  MinMaxValue next_dist_buffer;
+  double dist_from_ego_to_current_terminal_end{std::numeric_limits<double>::min()};
+  double dist_from_ego_to_current_terminal_start{std::numeric_limits<double>::min()};
```

Comment:
maybe the name can be shorter, maybe `dist_to_terminal_start`, I think unless otherwise specified it is implied that the distance is measured with respect to ego position. can just add a comment in the struct to describe it if someone is confused.